gui/install.sh

```diff
@@ -35,23 +37,42 @@ modify_locations() {	# Some files have placeholders for certain locations.  Modi
 }
 
 NEED_TO_UPDATE_HOST_NAME="true"
-
 CURRENT_HOSTNAME=`cat /etc/hostname | tr -d " \t\n\r"`
 
 # Check if the user is updating an existing installation.
 if [ "${1}" = "--update" -o "${1}" = "-update" ] ; then
-	UPDATE="true"
 	shift
+
+	echo -en '\n'
+	echo -e "*********************************************"
+	echo    "*** Performing update of the Allsky WebUI ***"
+	echo -e "*********************************************"
+	echo -en '\n'
+
 	if [ ! -d "${PORTAL_DIR}" ]; then
 		echo -e "${RED}Update specified but no existing WebUI found in '${PORTAL_DIR}'${NC}" 1>&2
 		exit 2
 	fi
 
 	modify_locations
+
+	# Add entries to sudoers file if not already there.
+	# This is only needed for people who updated allsky-portal but didn't update allsky.
+	# Don't simply copy the "allsky" file to /etc/sudoers.d in case "allsky" isn't up to date.
+	grep --silent "vcgencmd" /etc/sudoers.d/allsky
```

Comment:
Let's change this to `grep -c` (count matches) and capture the response.  Then in the conditional, ensure the response is 2, otherwise we need to update the sudoers file.

Reply:
Oh, I just noticed we only add the `/usr/bin` path now, not the `opt` path... 
  So, we need to check here for the correct path instead of just the presence of `vcgencmd`.